tests/ert3/console/integration/test_cli.py

```diff
@@ -217,7 +217,10 @@ def test_cli_status_some_runs(workspace, capsys):
     done_indices = [1, 3]
     for idx in done_indices:
         ert3.storage.init_experiment(
-            workspace=workspace, experiment_name=experiments[idx], parameters=[]
+            workspace=workspace,
+            experiment_name=experiments[idx],
+            parameters=[],
+            ensemble_size=-1,
```

Comment:
Here there is no data added to the experiment, so any size would do. I suggest we add any positive value. `42` is a good candidate 👍🏻

Reply:
This doesn't sound like a good approach. At least `-1` is obviously an invalid size value, while using `42` will make things difficult to debug in the long run.